crates/integration/src/tests.rs

```diff
@@ -234,6 +234,33 @@ fn signed_remainder() {
     run_differential(actions);
 }
 
+#[test]
```

Comment:
We can test this deferentially against geth EVM (see comment below). Then this manual test isn't needed :)

Could you please also remove the old test implementation that is commented out further down? Thanks!